plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouterArgs.sol

```diff
@@ -21,19 +21,29 @@ library PaymentStandardExitRouterArgs {
      * @notice Input args data for challengeStandardExit.
      * @param exitId Identifier of the standard exit to challenge.
      * @param outputType The output type of the exiting output.
-     * @param outputGuard The output guard of the output.
+     * @param exitingTx The transaction that is exiting.
      * @param challengeTxType The tx type of the challenge transaction.
      * @param challengeTx RLP encoded transaction that spends the exiting output.
      * @param inputIndex Which input of the challenging tx corresponds to the exiting output.
      * @param witness Witness data that can prove the exiting output is spent.
+     * @param spendingConditionOptionalArgs optional extra data for the spending condition.
+     * @param outputGuardPreimage (Optional) output guard preimage for the challenge tx to use the output
+     * @param challengeTxPos (Optional) tx position of the challenge tx if it is of MVP protocol.
+     * @param challengeTxInclusionProof (Optional) if the challenge tx is of MVP protocol provide the inclusion proof of it
+     * @param challengeTxConfirmSig (Optional) if the challenge tx is of MVP protocol provide the confirm sig of it
      */
     struct ChallengeStandardExitArgs {
         uint192 exitId;
         uint256 outputType;
-        bytes32 outputGuard;
+        bytes exitingTx;
         uint256 challengeTxType;
         bytes challengeTx;
-        uint8 inputIndex;
+        uint16 inputIndex;
```

Comment:
Why the increase to `uint16`?

Reply:
After this: https://github.com/omisego/plasma-contracts/issues/168, our `outputIndex` has bumped from uint8 to uint16. It is not necessary to bump for PaymentExit since we are bounded by `MAX_INPUT/OUTPUT_NUM` but it might be easier to keep consistent if possible.